code/game/machinery/constructable_frame.dm

```diff
@@ -199,11 +199,46 @@ micro-manipulator, console screen, beaker, Microlaser, matter bin, power cells.
 Note: Once everything is added to the public areas, will add m_amt and g_amt to circuit boards since autolathe won't be able
 to destroy them and players will be able to make replacements.
 */
+
+/obj/item/weapon/circuitboard/blank
+	name = "Unprinted Circuitboard"
+	desc = "A blank circuitboard ready for design."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "blank_mod"
+	var/datum/circuits/local_fuses = null
+
+/obj/item/weapon/circuitboard/blank/New()
+	..()
+	local_fuses = new(src)
+
+/obj/item/weapon/circuitboard/blank/attackby(obj/item/O as obj, mob/user as mob)
+	if(ismultitool(O))
+		var/boardType = local_fuses.assigned_boards["[local_fuses.localbit]"]

```

Comment:
No, the localbit is stored as a number but assigned boards is an associative list organized by strings.